src/commands/BootstrapCommand.js

```diff
@@ -201,6 +201,7 @@ export default class BootstrapCommand extends Command {
           const match = find(this.packages, { name: dependency});
           return match && filteredPackage.hasMatchingDependency(match);
         })
+        .map((dependency) => dependency.replace(/^(@.*\/)(.*)$/, "$2"))
```

Comment:
Hmm... we should be able to get the location from the package object, which we can grab from the depedency graph.  That's a little more future proof against [configurable package locations](https://github.com/lerna/lerna/issues/126), too.

If you drop this map you should be able to get the package using the dependency name:

```javascript
const dependencyPackage = this.packageGraph.get(dependency);
const dependencyLocation = dependencyPackage.location;
```